PipeMethodCalls/Endpoints/PipeServer.cs

```diff
@@ -32,11 +35,29 @@ public class PipeServer<THandling> : IPipeServer, IDisposable
 		/// <param name="handlerFactoryFunc">A factory function to provide the handler implementation.</param>
 		/// <param name="options">Extra options for the pipe.</param>
 		public PipeServer(IPipeSerializer serializer, string pipeName, Func<THandling> handlerFactoryFunc, PipeOptions? options = null)
+			: this(serializer, pipeName, handlerFactoryFunc, options, null)
+		{
+		}
+
+		/// <summary>
+		/// Initializes a new instance of the <see cref="PipeServer{THandling}"/> class.
+		/// </summary>
+		/// <param name="serializer">
+		/// The serializer to use for the pipe. You can include a library like PipeMethodCalls.NetJson and pass in <c>new NetJsonPipeSerializer()</c>.
+		/// This will serialize and deserialize method parameters and return values so they can be passed over the pipe.
+		/// </param>
+		/// <param name="pipeName">The pipe name.</param>
+		/// <param name="handlerFactoryFunc">A factory function to provide the handler implementation.</param>
+		/// <param name="options">Extra options for the pipe.</param>
+		/// <param name="maxNumberOfServerInstances">The maximum number of server instances that share the same name. You can pass <see cref="NamedPipeServerStream.MaxAllowedServerInstances"/> for this value,
+		/// defaults to <c>1</c>.</param>
+		public PipeServer(IPipeSerializer serializer, string pipeName, Func<THandling> handlerFactoryFunc, PipeOptions? options, int? maxNumberOfServerInstances)

```

Comment:
Can you add maxNumberOfServerInstances as another optional param that defaults to 1, instead of making another constructor overload?

int? maxNumberOfServerInstances = 1